easy-ui-react/src/ModeSwitcher/ModeSwitcher.module.scss

```diff
@@ -0,0 +1,29 @@
+@use "../styles/common" as *;
+@use "../styles/unstyled";
+
+.trigger {
+  @include unstyled.button;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: design-token("space.1");
+  padding: design-token("space.2");
+  border: design-token("shape.border_width.1") solid
+    theme-token("color.neutral.300");
+  border-radius: design-token("shape.border_radius.md");
+  cursor: pointer;
+  width: 100%;
+  max-width: 160px;
```

Comment:
set this as `max-width` as a magic number instead of `133px` because i noticed with `133px` the icon resizing caused the arrow to shrink with longer text content

Uploading Screen Recording 2024-12-17 at 5.50.09 PM.mov…